Doc/library/statistics.rst

```diff
@@ -596,14 +596,14 @@ However, for reading convenience, most of the examples show sorted sequences.
 
    Return the `Pearson's correlation coefficient
    <https://en.wikipedia.org/wiki/Pearson_correlation_coefficient>`_
-   for two variables. Pearson's correlation coefficient *r* takes values
+   for two inputs. Pearson's correlation coefficient *r* takes values
    between -1 and +1. It measures the strength and direction of the linear
    relationship, where +1 means very strong, positive linear relationship,
    -1 very strong, negative linear relationship, and 0 no linear relationship.
 
-   Raises :exc:`StatisticsError` if both variables have different number of data
-   points, or if any of the variables has less then two data points, or if
-   :func:`stdev` of any of the two variables is equal to zero (it is constant).
+   Both inputs must be of the same length (no less than two), and :func:`stdev`
+   of both inputs needs to be greater then zero, otherwise :exc:`StatisticsError`
```

Comment:
Perhaps just "and not be constant" rather than "and stdev of both inputs needs to be greater than zero"?